src/contractTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/IdamConsumerTestForPostUserInfo.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.wataskconfigurationapi;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.Pact;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslJsonRootValue;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.model.RequestResponsePact;
+import com.google.common.collect.Maps;
+import net.serenitybdd.rest.SerenityRest;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IdamConsumerTestForPostUserInfo extends SpringBootContractBaseTest {
+
+    private static final String IDAM_OPENID_USERINFO_URL = "/o/userinfo";
+
+    @Pact(provider = "Idam_api", consumer = "wa_task_management_api")
```

Comment:
Please reference the following ia-case-api branch "ConsumerContractTests".
For example;
In @Pact(provider = "Idam_api", consumer = "wa_task_management_api"), Idam_api should be idamApi_oidc